datastream-server/src/main/java/com/linkedin/datastream/server/Coordinator.java

```diff
@@ -819,7 +819,7 @@ private void initializeTask(DatastreamTask task, Set<DatastreamTask> failedDatas
       Map<Integer, String> checkpoints = producer.loadCheckpoints(task);
       taskImpl.setCheckpoints(checkpoints);
     } catch (Exception e) {
-      _log.warn("Failed to initialize {} task", task.getDatastreamTaskName());
+      _log.warn("Failed to initialize task: " + task.getDatastreamTaskName(), e);
```

Comment:
Can you change the logger level as well ?

Reply:
Changed the log level to error